modules/core/src/main/java/org/apache/ignite/internal/pagemem/impl/PageMemoryNoStoreImpl.java

```diff
@@ -322,9 +320,6 @@ public PageMemoryNoStoreImpl(
                 "  ^-- Enable eviction or expiration policies"
             );
 
-            if (ctx != null)
-                ctx.kernalContext().failure().process(new FailureContext(FailureType.CRITICAL_ERROR, oom));
```

Comment:
Field "ctx" is never used now. Could you remove it?

Reply:
done